Add trans for Width and Height field on MediaAction.php

LikeLown opened the pull request:
Add trans for Width and Height field.

LikeLown committed:
Update MediaAction.php

Add trans for Width and Height field.

awcodes commented:
Can you add these to the media-modal file instead of reusing the oembed-modal, please?

awcodes commented:
I think you're missing some of the styles, make sure you have included the views and the stylesheet in a custom theme according to the installation setups in the readme.

This is what I'm currently seeing with your PR. If you can resolve this I'll merge it in.

Also think the mobile looks weird without the borders, can you make it so the styling is only applied when it's in the 'table' view?

![Screenshot 2024-11-01 at 10 31 31 AM](https://github.com/user-attachments/assets/59d891c0-7126-4ca1-adcc-2e2efd7e4940)

![Screenshot 2024-11-01 at 10 31 42 AM](https://github.com/user-attachments/assets/74316ef4-6085-40dc-befb-cd58eebac7a8)

LikeLown committed:
modified:   resources/lang/en/media-modal.php
	modified:   resources/lang/fr/media-modal.php
	modified:   src/Actions/MediaAction.php

awcodes commented:
Thank you. 😀
